src/Inventory/Inventory.js

```diff
@@ -38,12 +53,17 @@ export default class Inventory extends Component {
 			headerTitleStyle: {
 				color: colors.tint,
 			},
+			// save button for editing?
 			headerLeft: (
-				<Button 
-					title='Cancel'
-					onPress={() => navigation.goBack()}
-					backgroundColor={colors.background}
-				/>
+				<View style={stylesheet.leftButton}>
```

Comment:
Just a thought; maybe at some point we should do some refactoring and pull this whole left header out and reuse it on different components